### ISSUE TITLE
Error when attempting to update from bundle 2.x to 3.x

### ISSUE THREAD

**Issue by aaronmanill** [Azure/azure-functions-core-tools#2976]
My func host start is failing with this message
[2022-03-04T17:55:46.944Z] A host error has occurred during startup operation 'f895a823-a1a8-4576-9616-8e114f1789f4'.
[2022-03-04T17:55:46.945Z] System.Private.CoreLib: Could not load file or assembly 'C:\Users\---\.azure-functions-core-tools\Functions\ExtensionBundles\Microsoft.Azure.Functions.ExtensionBundle\3.3.0\bin_v3\linux-x64\Microsoft.Azure.WebJobs.Extensions.DurableTask.dll'. Format of the executable (.exe) or library (.dll) is invalid.
Value cannot be null. (Parameter 'provider')

extensions.csproj:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
	<WarningsAsErrors></WarningsAsErrors>
	<DefaultItemExcludes>**</DefaultItemExcludes>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.6.1" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="5.2.0" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.SignalRService" Version="1.6.0" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.WebPubSub" Version="1.0.0" />
    <PackageReference Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="1.1.3" />
  </ItemGroup>
</Project>

```
host.json 
```
{
  "version": "2.0",
  "logging": {
    "applicationInsights": {
      "samplingSettings": {
        "isEnabled": true,
        "excludedTypes": "Request"
      }
    }
  },
  "extensionBundle": {
    "id": "Microsoft.Azure.Functions.ExtensionBundle",
    "version": "[3.3.0, 4.0.0)"
  }
}
```

**Comment by michaelpeng36** [comment]
@aaronmanill Could you clarify why you are using a specific version of the DurableTask extension? You may need to exclusively use the Extensions bundle (without specifying specific versions in `extensions.csproj`) or remove the Extensions bundle dependency entirely, as this may be the result of versioning conflicts. @davidmrdavid, could you clarify if this is the appropriate version of the DurableTask extension for ExtensionBundle 3.3.0?

**Comment by aaronmanill** [comment]
It looks like extensions.csproj is supposed to not exist when I use the bundle, I had been using it for WebPubSub development, and was mistakenly told that PubSub was part of the 3.3.0 package now and forgot to delete extensions.csproj before putting the bundle back, but WebPubSub was not part of 3.3.0 so I'm resolved and waiting for a 3.4.0 release
